submissions/finalise.R

```diff
@@ -122,13 +122,15 @@ if (length(crossing_locations) > 0) {
 # Add cumulative forecast -------------------------------------------------
 # get cumulative data
 source(here("utils", "get-us-data.R"))
-cumulative_state <- setDT(get_us_deaths(data = "cumulative"))
-cumulative_state <- 
-  cumulative_state[date == min(as.Date(submission$target_end_date)) - weeks(1),
+cumulative <- setDT(get_us_data(data = "deaths",
+                                      include_national = TRUE,
+                                      incident = FALSE))
+cumulative$deaths <- cumulative$value
```

Comment:
as dt for style reasons it would be nice to do the variable updating in a dt like way but no functional difference.